src/main/java/controller/Condition.java

```diff
@@ -0,0 +1,5 @@
+package main.java.controller;

```

Comment:
Enum is not a controller, so should not be here. Either move to .model package or create an enums package.